src/commands/login.rs

```diff
@@ -260,6 +272,16 @@ impl LoginCommand {
         self.save_login_info(&login_connection)
     }
 
+    async fn login_using_fermyon_token(&self, token: &str) -> Result<()> {
```

Comment:
This should verify the token before saving. I don't see a dedicated "user info" API but it could just check the apps list.

Reply:
done